packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -723,6 +775,13 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
     }
   }
 
+  private _removeSelectedIndex(index: number) {
+    const selectedIndices = this.state.selectedIndices && this.state.selectedIndices.filter((idx: number) => idx !== index) || [];
+    this.setState({
+      selectedIndices: selectedIndices
```

Comment:
Shouldn't this call onChanged?

Reply:
This is not needed, because _setSelectedIndex() will call onChanged() CB.

Reply:
Where is setSelectedIndex being called?

Reply:
Good question. I looked into further. onChanged is only triggered for mouse click, but not for ENTER key. Fixed it now.